### PR TITLE
Hide faucet on prod

### PR DESCRIPTION
**Pull request by RamsesDijkstra**
I'm assuming this is enough? I was unable to test this, when i go to staking overview it complains that i have to switch to arbitrum network. Which the account i signed on with, is currently on.
So i've decided to just PR this, if it isn't sufficient then let me know

### CODE REVIEW

File: packages/app/src/scenes/widgets/pages/StakingViewWidget/components/MyWallet/MyWallet.tsx [hunk 1]
```diff
@@ -106,26 +108,31 @@ const MyWallet: FC<PropsInterface> = ({
           />
         </styled.RewardsContainer>
 
-        <styled.Title>Use Faucet</styled.Title>
-        <styled.AirdropContainer>
-          {canRequestAirdrop !== false ? (
+        {appVariables.CONTRACT_FAUCET_ADDRESS !== "" ?? (
```
Comment (dmitry-yudakov) [inline packages/app/src/scenes/widgets/pages/StakingViewWidget/components/MyWallet/MyWallet.tsx]:
```suggestion
        {!!appVariables.CONTRACT_FAUCET_ADDRESS && (
```
This is a bit clearer to me :)
Using the double negation here means that if it's not `(empty string, null or undefined)`, we'll render the Faucet components.

File: packages/app/src/scenes/widgets/pages/StakingViewWidget/components/MyWallet/MyWallet.tsx [hunk 2]
```diff
@@ -8,6 +8,8 @@ import {WalletModelInterface} from 'core/models';
 import {useBlockchain} from 'shared/hooks';
 import {TokenSelector} from 'ui-kit';
 
+import {appVariables} from "../../../../../../api/constants";
```
Comment (dmitry-yudakov) [inline packages/app/src/scenes/widgets/pages/StakingViewWidget/components/MyWallet/MyWallet.tsx]: We use `baseUrl` in `tsconfig.json` so it should work as just `"api/constants"` without going to the X-th floor :)

**Review by dmitry-yudakov — CHANGES_REQUESTED** [review]

**Review by jor-rit — APPROVED** [review]
yes, with @dmitry-yudakov comment, makes it more readable and 'safe' for whatever weird gets set (or not set) in the config

File: packages/app/src/scenes/widgets/pages/StakingViewWidget/components/MyWallet/MyWallet.tsx (hunk 2 above)
Comment (RamsesDijkstra) [reply packages/app/src/scenes/widgets/pages/StakingViewWidget/components/MyWallet/MyWallet.tsx]: seems like my ide was being naughty

**Review by dmitry-yudakov — APPROVED** [review]

**Pull request merged**